static/src/stylesheets/module/identity/_identity.scss

```diff
@@ -386,15 +386,39 @@
     &.is-updating-subscriptions {
         display: inherit;
         margin-top: 0;
+        font-size: 0;
         background-color: $neutral-5;
         background-repeat: no-repeat;
         background-position: center;
         background-size: 40px;
     }
+
+    &.email-unsubscribe {
+        float: none;
+        color: $guardian-brand;
+        background-color: #ffffff;
+        border: 2px solid $guardian-brand;
+        min-width: $gs-baseline * 14;
```

Comment:
Better to user a `gs-span()` for widths than a large multiple `$gs-baseline` (likewise in `.save__button` on line 421)